### PR TITLE
PID for SemarmeHID

### PR DESCRIPTION
**Pull request by carakatama**
PID: BADE

### CODE REVIEW

File: org/semarme/1209/BADE/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (Arachnid) [inline org/semarme/1209/BADE/index.md]: This file needs to be in /1209/BADE/index.md, not under your org directory.

Comment (carakatama) [inline org/semarme/1209/BADE/index.md]: OK, I have been change it

File: 1209/bade/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: SemarmeHID
+owner: semarme
+license: GPL V2.0
+site: http://github.com/carakatama/SemarmeHID/
+source: http://github.com/carakatama/SemarmeHID/
+---
```
Comment (Arachnid) [inline 1209/bade/index.md]: Please add a short description of your device here.

Comment (carakatama) [inline 1209/bade/index.md]: now i did it

**Pull request merged**